frontend/src/components/communication/chat/ChatPreviews.js

```diff
@@ -111,7 +102,7 @@ export default function ChatPreviews({ chats, loadFunc, hasMore }) {
           chat={chat}
         />
       ))}
-      {loadingSpinner()}
+      <LoadingSpinner />
```

Comment:
We should pass the `isLoading` prop here I think
```suggestion
      <LoadingSpinner isLoading={isLoading}/>
```